README.md

```
@@ -1,8 +1,159 @@
+✗ Metal acceleration not available
+✓ CPU cores available: 12
+
+Selected acceleration: CUDA
+Loading Llama-3.2-1.3B...
+Memory Allocated: 1.3GB VRAM
+```
+
+
+## 🗃️ Project Structure
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)